public/apps/account/__snapshots__/account-nav-button.test.tsx.snap

```diff
@@ -59,6 +59,7 @@ exports[`Account navigation button renders 1`] = `
               />
             </EuiListGroup>
             <EuiListGroupItem
+              color="subdued"
```

Comment:
Is this intentional? Looks unrelated to the topic of this change.

Reply:
Yep this is to fix a snapshot test.